weather-forecast/ecrad-3d-correction/unets/ci/requirements_data.txt

```diff
@@ -0,0 +1,7 @@
+climetlab==0.20.1
```

Comment:
I think we don't need this lib, no ? It is not imported in the `generate_synthetic_data.py` file

Reply:
The "problem" is that the class in generate_synthetic.py inherits from the one in dataproc.py and overrides just the download methods. Now, when I run the process method (the one of dataproc.py)  climatlab is seen as a dependency due to the import command.